### DISCUSSION TITLE
Edit your pacman.conf

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1367] [Tips]
Users updating their system regularly via `pacman -Syu` (or `yay -Syu`) noticed no available updates for about one month. 
Reason:
https://archlinuxarm.org/forum/viewtopic.php?f=57&t=16491#p71258

Now updates are back again, but you have to edit your `/etc/pacman.conf` as described here:
https://archlinux.org/news/git-migration-completed
Quote:
_Update your system and merge the pacman pacnew `/etc/pacman.conf.pacnew` file. This is required as we have moved the [community] repository into [extra]._
TL;DR:
Remove the community repository from `/etc/pacman.conf`.

**Comment by R101G** [comment]
Thanks for the heads-up @klassikmann. I was wondering why it was so quiet.
